src/catalog/catalog.h

```diff
@@ -54,6 +65,8 @@ class Catalog {
   bool addTable(TableSchema t);
   bool addTable(TableSchema t, std::shared_ptr<DBTable> table_ref);
 
+  bool clearMemTable(std::string name);
```

Comment:
minor: The name of the function could be confusing, clear a table could mean that you erase its contents rather than deleting an entire table.

Reply:
renamed it to "dropMemTable"